pkg/reconciler/gc/v2/gc.go

```diff
@@ -61,6 +61,10 @@ func Collect(
 	})
 
 	for _, rev := range revs[gcSkipOffset:] {
+		preserve := rev.ObjectMeta.Annotations[serving.RevisionPreservedAnnotationKey]
```

Comment:
no need for the var here
`if rev.ObjectMeta.Annotations[serving.RevisionPreservedAnnotationKey] == "true"`

Reply:
Also the canonical in knative version is `strings.EqualFold(val, "true")`.